src/Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.Core/Services/DevCanvasResultSourceService.cs

```diff
@@ -0,0 +1,384 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.Caching;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+
+using CSharpFunctionalExtensions;
+
+using EnvDTE;
+
+using EnvDTE80;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.Sarif.Viewer;
+using Microsoft.Sarif.Viewer.ResultSources.Domain;
+using Microsoft.Sarif.Viewer.ResultSources.Domain.Abstractions;
+using Microsoft.Sarif.Viewer.ResultSources.Domain.Models;
+using Microsoft.Sarif.Viewer.ResultSources.Domain.Services;
+using Microsoft.Sarif.Viewer.Shell;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Threading;
+
+using Newtonsoft.Json;
+
+using Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.Core.Models;
+
+using Result = CSharpFunctionalExtensions.Result;
+
+namespace Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.Core.Services
+{
+    public class DevCanvasResultSourceService : IResultSourceService
```

Comment:
doc comment

Reply:
Still don't see one here.

Reply:
<img width="338" alt="image" src="https://github.com/microsoft/sarif-visualstudio-extension/assets/126511432/f6153dd7-43d8-431c-b9b7-006c3aeb7027">
one of the things I hate w/ github is that it doesn't seem to really update the preview window with the new content like in ADO